PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -169,9 +175,10 @@ private void sendPayPalRequest(
                         null,
                         new SettingsConfig(),
                         new UIConfig(
-                                false
+                            false
                         ),
-                        payPalRequest.getReturnUrl()
+                        payPalRequest.getReturnUrl(),
+                        authConfig
```

Comment:
We we OK to pass in a null `authConfig` if email isn't set?

Reply:
Yes we handle nullability in the SDK if authConfig is null